### PR TITLE
Support for OneTouch Ultra2

### PR DESCRIPTION
**Pull request by gniezen**
We've had the Ultra2 driver in the codebase for a while now, but it has always been disabled since it didn't have proper error handling and didn't adhere to the UTC bootstrapping time format.

This PR adds:
- TZOUtil across-the-the-board time
- units from meter, not hard-coded
- BGM checklist
- skipping control solution tests
- annotating HI/LOs

### CODE REVIEW

**Comment by gniezen** [comment]
[OneTouch Ultra 2 and OneTouch Mini](https://trello.com/c/drBwiv9K/198-onetouch-ultra-2-and-onetouch-mini)

File: lib/drivers/onetouch/oneTouchUltra2.js [hunk 1]
```diff
@@ -213,43 +199,73 @@ module.exports = function (config) {
      }
 
     // get data from lines
-    var dpattern = /P "(\w+)","(\d{2}\/\d{2}\/\d{2})","(\d{2}:\d{2}:\d{2})   ","..(\d{3}).","(.)","(\d{2})",\W00\W(.+)/;
+    var dpattern = /P "(\w+)","(\d{2}\/\d{2}\/\d{2})","(\d{2}:\d{2}:\d{2})   ","(..\d{3}).","(.)","(\d{2})",\W00\W(.+)/;
```
Comment (DorianScholz) [inline lib/drivers/onetouch/oneTouchUltra2.js]: The regex for the glucose value does not capture the optional trailing question mark which signals a parity error according to the specs. Is that intentional, or should there be some specific handling of values with this flag?

Comment (gniezen) [inline lib/drivers/onetouch/oneTouchUltra2.js]: True, it's currently ignoring the parity error flag. I think the correct response would be to drop the reading if there is a parity error. I'll update this right now.

**Comment by gniezen** [comment]
You're right, line 356-358 should not be there.

**Comment by DorianScholz** [comment]
The two new commits look good to me.
From my perspective of a visual code review, this PR is ready to be merged.
Should I approve it, or leave that to someone that has actually tested it against the device?

**Comment by gniezen** [comment]
You can approve. I will then merge it into master and create a new version of the dev uploader for Eric or Nick to test before we push to production.

**Comment by gniezen** [comment]
@pazaan Dorian has already reviewed the driver, but I have made some additional changes in commit b03ba87 after I discovered that sometimes it only uploaded around 12 out of 20 records. Can you please review the additional changes? (I also need someone with write access to approve)

**Pull request merged**